spring-pulsar/src/main/java/org/springframework/pulsar/listener/PulsarContainerProperties.java

```diff
@@ -246,4 +257,35 @@ public void setPulsarConsumerProperties(Properties pulsarConsumerProperties) {
 		this.pulsarConsumerProperties = pulsarConsumerProperties;
 	}
 
+	@SuppressWarnings("unchecked")
+	public void updateContainerProperties() {
+		if (!this.pulsarConsumerProperties.isEmpty()) {
```

Comment:
**[NIT]** An invert of this if and short-circuit return will allow all the code below to shift over.